docs/whitepaper.tex

```diff
@@ -157,80 +157,81 @@ \section{Implementation details}
   When providing liquidity, $x \rightarrow x'$ and $y \rightarrow y'$ has the
   effect that $k \rightarrow k'$. The number of liquidity tokens the LP
   receives, $\Delta L$, then follows the rule $\Delta L / L = \Delta k / k$.
-}, we need a liquidity token to exist, with a minting policy linked to a
-specific pool.
-Secondly, as in the Cardano Application Template, we want each pool be
-identified by an NFT minted through a specific minting policy. This allows
-anyone to verify that a UTXO at the pool script address is valid, as well as to
-trace each pool.
-Finally, our protocol contains a number of cyclic dependencies between the
-different scripts. In order to resolve this, we follow the Config UTXO pattern.
+}, we need a liquidity token linked to the respective pool to exist. 
+This implies that the liquidity token needs to be minted/ burnt using a minting policy linked to a specific pool.
+Secondly, as in the Cardano Application Template, we want the liquidity pools to be
```

Comment:
What are "liquidity pools" here? As opposed to what other type of pool?

Reply:
but to be consistent with the rest of the text I'll remove liquidity